cmd/kanikoExecute.go

```diff
@@ -25,22 +25,15 @@ type kanikoHttpClient interface {
 }
 
 func installSyft(shellRunner command.ShellRunner, fileUtils piperutils.FileUtils, httpClient kanikoHttpClient) error {
-	installationScript := "./install.sh"
-	err := httpClient.DownloadFile(syftURL, installationScript, nil, nil)
+	//Pinning to latest version as of today , to avoid nasty surprises
+	err := httpClient.DownloadFile("https://github.com/anchore/syft/releases/download/v0.60.3/syft_0.60.3_linux_amd64.tar.gz", "syftBinary.tar.gz", nil, nil)
```

Comment:
Can we move this url to const at the very begining of the file? Once we decide to update syft's version it'll be easier to find it.

Reply:
Good catch.Done